examples/hosts/app-integration/external-controller/package.json

```diff
@@ -33,22 +33,15 @@
     "webpack:dev": "webpack --env.development"
   },
   "dependencies": {
-    "@fluid-experimental/data-objects": "^0.40.0",
-    "@fluid-experimental/fluid-static": "^0.40.0",
+    "fluid-framework": "^0.40.0",
     "@fluid-experimental/tinylicious-client": "^0.40.0",
-    "@fluidframework/aqueduct": "^0.40.0",
-    "@fluidframework/container-definitions": "^0.40.0",
-    "@fluidframework/container-loader": "^0.40.0",
-    "@fluidframework/container-runtime-definitions": "^0.40.0",
-    "@fluidframework/map": "^0.40.0",
-    "@fluidframework/request-handler": "^0.40.0",
-    "@fluidframework/runtime-utils": "^0.40.0",
     "css-loader": "^1.0.0",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
     "style-loader": "^1.0.0"
   },
   "devDependencies": {
+    "@fluid-experimental/fluid-static": "^0.40.0",
```

Comment:
@micahgodbolt we should think about how developers test with this model.

Reply:
this dev dep is only there due to using fluid-static in existing test code. I don't think this is going to be a long term solution.